### DISCUSSION TITLE
Devtools and two databases (Mysql and MongoDB)

### DISCUSSION THREAD

**Discussion by ltchurik** [Laravel-Backpack/community-forum#1084] [Q&A (Help)] [answered]
Hello,
 i am using mysql and mondgo db in my project. When i added mongodb thru composer issues begin. Errors begin to appera when accessing devtools. I fuguread out that i can hide mongo models in different directory and configure devtools so they dont see it , 1 problem fixed. But when i try to create model thru gui i get:

BadMethodCallException
Method MongoDB\Laravel\Schema\Grammar::compileColumns does not exist.

I am thinking that somehow mongo eloquent became default for devtools or something like that. Can someone guide me how to fix this? Thanks

**Comment by pxpm** [comment, marked as answer]
Hey @ltchurik 👍 

There are some known issues with mongodb and we don't have yet proper path to follow regarding that nor a time-frame I can give you to work on that. Supporting mongodb is not in our priority list. 

Devtools uses the `config('database.default')` connection, so if you can change your default connection to the mysql it should work since you already hidden the models.

If that's not possible, what I would suggest is that you create a dummy project with mysql or other relational database and use devtools in that project to generate the files, then copy over the generated files to your multi db app, and change the `extends Model` etc for models you created for mongo. 

Sorry I can't help much more here.

Cheers
